packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -512,7 +512,7 @@ export const FieldStruct: Describe<FieldElement> = element('Field', {
  */
 export const FormChildStruct = children(
```

Comment:
Can we remove `FormChildStruct` and just use `children(BoxChildStruct)` for `FormStruct`?

Reply:
Or alternatively to not break as much:
`export const FormChildStruct = BoxChildrenStruct`;